### DISCUSSION TITLE
A favorite list?

### DISCUSSION THREAD

**Discussion by Njasa2k** [civitai/civitai#38] [Ideas]
I was looking for a way to save models that I like and many sites have a favorite list that makes it easy to save.

**Comment by JustMaier** [comment]
Maybe a list of models that I've Downloaded as well so that I can make sure to go back and review them.

**Comment by JustMaier** [comment]
**This is live!**

Favorite counts show up on the main page:
![image](https://user-images.githubusercontent.com/607609/204674436-7ba39e9c-7f97-4a0a-afca-d3f3cda7e512.png)

You can toggle your favorites:
![image](https://user-images.githubusercontent.com/607609/204674502-1eb4b53a-60a2-429a-a114-98a27392e1a8.png)
![image](https://user-images.githubusercontent.com/607609/204674557-f39929d6-abab-4d87-9dfd-e8893e9d3c2d.png)

And you can come back to your favorites at any time:
![image](https://user-images.githubusercontent.com/607609/204674653-641a0449-13fb-46fb-b0cc-3a5a5ac7d8f7.png)
